suite-native/module-onboarding/src/screens/OnboardingAssets.tsx

```diff
@@ -0,0 +1,149 @@
+import { View } from 'react-native';
+import { useDispatch } from 'react-redux';
+
+import { CryptoIcon, Icon } from '@trezor/icons';
+import TrezorConnect, { AccountInfo } from '@trezor/connect';
+import { Screen, StackProps } from '@suite-native/navigation';
+import {
+    Box,
+    Button,
+    Text,
+    Card,
+    InputWrapper,
+    Input,
+    SelectableListItem,
+    VStack,
+} from '@suite-native/atoms';
+import { prepareNativeStyle, useNativeStyles } from '@trezor/styles';
+
+import { OnboardingStackParamList, OnboardingStackRoutes } from '../navigation/routes';
+import { AssetsLoader } from '../components/AssetsLoader';
+import { setOnboardingFinished } from '../slice';
+
+const assetsStyle = prepareNativeStyle(utils => ({
+    flex: 1,
+    padding: utils.spacings.medium,
+    justifyContent: 'space-between',
+}));
+
+export const OnboardingAssets = ({
+    route,
+}: StackProps<OnboardingStackParamList, OnboardingStackRoutes.OnboardingAssets>) => {
+    const dispatch = useDispatch();
+    const [radioChecked, setRadioChecked] = useState<string>('firstSelectable');
+    const [inputText, setInputText] = useState<string>('');
+    const [accountInfo, setAccountInfo] = useState<AccountInfo | null>(null);
+    const { applyStyle } = useNativeStyles();
+
+    const { xpubAddress, currencySymbol } = route.params;
+
+    const getAccountInfo = useCallback(() => {
+        TrezorConnect.getAccountInfo({
+            coin: currencySymbol,
+            descriptor: xpubAddress,
+        })
+            .then(accountInfo => {
+                if (accountInfo?.success) {
+                    setAccountInfo(accountInfo.payload);
+                }
+                // TODO error?
+            })
+            .catch(error => {
+                // eslint-disable-next-line no-console
+                console.log('getAccountInfo failed: ', JSON.stringify(error));
+            });
+    }, [xpubAddress, currencySymbol]);
+
+    useEffect(() => {
+        const fetchingTimerId = setTimeout(() => {
+            getAccountInfo();
+        }, 800);
+
+        return () => clearTimeout(fetchingTimerId);
+    }, [getAccountInfo]);
+
+    const handleConfirmAssets = () => {
+        dispatch(setOnboardingFinished(true));
+    };
+
+    const handleRadioPress = (value: string | number) => {
+        setRadioChecked(value.toString());
+    };
+
+    return (
+        <Screen>
+            {!accountInfo ? (
+                <AssetsLoader />
+            ) : (
+                <View style={[applyStyle(assetsStyle)]}>
+                    <View>
+                        <Box flexDirection="row" justifyContent="space-between">
+                            <Text variant="titleMedium" color="black">
+                                Import assets
+                            </Text>
+                            <Icon name="close" />
+                        </Box>
+                        <Card>
+                            <Box marginTop="large" marginBottom="medium">
```

Comment:
I would probably make this separate component.

Reply:
Good point.

Reply:
https://github.com/trezor/trezor-suite/pull/5869/commits/29a870306e3ded7949d89c59eeaa4b8ee857743c